packages/apidom-ns-workflows-1/src/elements/nces/Criteria.ts

```diff
@@ -0,0 +1,12 @@
+import { ArrayElement, Attributes, Meta } from '@swagger-api/apidom-core';
+
```

Comment:
I would suggest the following definition:

```ts
class SuccessActionCriteria extends ArrayElement {
  static primaryClass = 'success-action-criteria';

  constructor(content?: Array<unknown>, meta?: Meta, attributes?: Attributes) {
    super(content, meta, attributes);
    this.classes.push(SuccessActionCriteria.primaryClass);
    this.classes.push('criteria');
  }
}

export default SuccessActionCriteria;
```

### Reasoning

When we're at `Failure Action Object` we'll have to define `FailureActionCriteria` NCE. It's best to have two NCEs definition to be able to distinguish where this criteria array comes from. On top of that it will be assigned `criteria` meta class as well to be able to work with `SuccessActionCriteria` and `FailureActionCriteria` in polymorphic way if needed.